datahub-classify/tests/test_similarity_predictor.py

```diff
@@ -0,0 +1,487 @@
+import glob
+import itertools
+import json
+import logging
+import os
+import pickle
+import time
+from typing import Dict, Tuple
+
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.metrics import classification_report, confusion_matrix
+
+from datahub_classify.helper_classes import (
+    ColumnInfo,
+    ColumnMetadata,
+    TableInfo,
+    TableMetadata,
+)
+from datahub_classify.similarity_predictor import check_similarity
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
this also isn't needed - pytest will set things up

Reply:
Removed from code